services/brig/src/Brig/API/User.hs

```diff
@@ -335,7 +328,7 @@ createUser new = do
 
     wrapClient $ Data.insertAccount account Nothing pw False
     liftSem $ GalleyAPIAccess.createSelfConv uid
-    liftSem $ Intra.onUserEvent uid Nothing (UserCreated (accountUser account))
+    liftSem $ Events.generateUserEvent uid Nothing (UserCreated (accountUser account))
```

Comment:
I think `activateUser` does that part. 

The reason for pulling this out was that we were processing notifications meant for users as if they were intra backend communication which didn't seem very intuitive. Yes, this opens a room for bugs, but we have to trust our testing otherwise refactorings like this one would never happen.